collector/benchmarks/README.md

```diff
@@ -68,6 +68,8 @@ programs.
   caused [poor performance](https://github.com/rust-lang/rust/issues/32278) in
   the past.
 - **ctfe-stress-2**: A stress test for compile-time function evaluation.
+- **ctfe-stress-uninit**: A stress test for representation of values with
+  uninitialized bytes in compile-time function evaluation.
```

Comment:
This should be updated.

Reply:
Remove it or merge the two comments?

Reply:
There is no uninit benchmark anymore, I would expect a single entry with -3.

Reply:
Probably just dropping the uninit bit; that can be moved to a comment in the file.